data/assets/scene/solarsystem/missions/artemis/trail.asset

```diff
@@ -0,0 +1,88 @@
+local earthTransforms = asset.require("scene/solarsystem/planets/earth/transforms")
+local moonTransforms = asset.require("scene/solarsystem/planets/earth/moon/moon")
+asset.require("spice/base")
+
+local kernels = asset.syncedResource({
+  Name = "Artemis Kernels",
+  Type = "HttpSynchronization",
+  Identifier = "artemis_kernels",
+  Version = 1
+})
+
+local ArtemisEarthTrail = {
+  Identifier = "ArtemisEarthTrail",
+  Parent = earthTransforms.EarthCenter.Identifier,
+  Renderable = {
+    Type = "RenderableTrailTrajectory",
+    Translation = {
+      Type = "SpiceTranslation",
+      Target = "-1023", -- Artemis-1
+      Observer = "EARTH",
+      Frame = "GALACTIC",
+      Kernels = { kernels .. "artemis.bsp" }
+    },
+    Color = { 0.2, 0.78, 0.635 },
+    Fade = 7.0,
+    StartTime = "2022-Nov-16 08:19:00.000",
+    EndTime = "2022-Dec-11 17:20:45.065",
```

Comment:
Technically these should be:
YYYY MON DY .... format;

so:
"2022 NOV 16 08:19:00"
and
"2022 DEC 11 17:20:45.065"